docs/data_sources/future_data.rst

```diff
@@ -31,16 +31,6 @@ integration, don't hesitate to `open an issue on Github
 <https://github.com/catalyst-cooperative/pudl/issues>`__ requesting the data and
 explaining why it would be useful.
 
-.. _data-eia176:
-
-EIA Form 176
-^^^^^^^^^^^^
-
-EIA `Form 176 <https://www.eia.gov/dnav/ng/TblDefs/NG_DataSources.html#s176>`__, also
-known as the **Annual Report of Natural and Supplemental Gas Supply and Disposition**,
-describes the origins, suppliers, and disposition of natural gas on a yearly and state
-by state basis. EIA Forms 191 and 757a come in the same bulk data download and also
-pertain to natural gas.
```

Comment:
No longer "Future Data" (!!)

Removing the anchor to this header didn't break anything anywhere else in the docs so I don't think we were linking to it previously.